tabpy-server/tabpy_server/state.ini

```diff
@@ -11,5 +11,4 @@ Access-Control-Allow-Methods =
 [Query Objects Docstrings]
 
 [Meta]
-Revision Number = 1
-
+Revision Number = 1
```

Comment:
To restore `state.ini` or any other file reverting all the changes run `git checkout HEAD -- tabpy-server/tabpy_server/stati.ini`